scripts/binary_no_descartes_robot_motion.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
```

Comment:
*Note for this entire file*
Ideally in the future.. Keep code modular and only put color specific variations into their own file or just use an `if ___` toggle in the main script.
Fine for this PR, but something to fix in the future. Making note here to remember.